SLHCUpgradeSimulations/Configuration/python/phase1TkCustoms.py

```diff
@@ -427,3 +638,32 @@ def customise_Reco(process,pileup):
     process.preDuplicateMergingDisplacedTracks.trackProducers.remove("muonSeededTracksInOut")
 
     return process
+
+def customise_Reco_Run2(process):

```

Comment:
Run2 spans move to Phase1. So, better not introduce much confusion. Maybe _Run2_Legacy and Run2_Phase1